composer.json

```diff
@@ -22,7 +22,9 @@
     "require-dev": {
         "guzzlehttp/guzzle": "^6.2",
         "symfony/framework-bundle": "^2.7|^3.0",
-        "symfony/yaml": "^2.7|^3.0"
+        "symfony/yaml": "^2.7|^3.0",
+        "doctrine/common": "^2.7",
```

Comment:
LOL, shit on that election. It's not like we have a composer.lock file, you won't break anything by manually editing this file.